doc/source/whatsnew/v1.1.0.rst

```diff
@@ -224,6 +224,9 @@ Performance improvements
 - The internal index method :meth:`~Index._shallow_copy` now copies cached attributes over to the new index,
   avoiding creating these again on the new index. This can speed up many operations that depend on creating copies of
   existing indexes (:issue:`28584`, :issue:`32640`, :issue:`32669`)
+- Performance improvement when creating sparse :class:`DataFrame` from
```

Comment:
Typically we phrase this as

```suggestion
- Performance improvement when creating a :class:`DataFrame` with sparse values
```

to avoid confusion with the old SparseDataFrame.